### DISCUSSION TITLE
Is there a print plugin?

### DISCUSSION THREAD

**Discussion by kivicms** [bcakmakoglu/vue-flow#1092] [Q&A] [answered]
How to print flow?

**Comment by bcakmakoglu** [comment, marked as answer]
Regardless of what you actually "mean" by print - no, there are no "plugins" as of yet.

Furthermore, you'll have to be a bit more specific as I'm not (yet) a mind-reader 😉 
What do you mean by "print"? A screenshot?

There are libraries like `html-to-image` which you can use for such purposes.
